src/browser/components/events/event_view.jsx

```diff
@@ -20,17 +20,18 @@ EventView = React.createClass({
         if (event) {
             return (
                 <div className='event-view-container'>
-                    <h1 className='event-view-name'>{event.name}</h1>
-                    <div>
-                        <span className='icon icon-location' />
-                        <h2 className='event-view-address'>{event.address}</h2>
+                    <div className='event-view-info'>
```

Comment:
This was an interesting one : This was the reason why your "Rides" section couldn't expand to the full width. You need to pull the event-view-info into a div and put margins on that instead of the whole parent div. Otherwise, you'd have to put negative margins on your Rides component - no bueno